### DISCUSSION TITLE
SOC: Request failed with status code 500

### DISCUSSION THREAD

**Discussion by mrocaruano** [Security-Onion-Solutions/securityonion#4999] [Unsupported Versions]
![image](https://user-images.githubusercontent.com/43893495/127570846-e07bb065-d2b3-47ff-b479-d46d919f41d9.png)

I deployed Security Onion 2.3.61. I keep getting this Request Failed with status code 500 error. so-status shows everything is okay. I have tried increasing the easheap on manager and elasticsearch under /opt/so/saltstack/local/pilla/minions/securityonion.sls to see if that would solve the issue and no luck. I also copied the nginx.conf file from older versions of security onion (2.3.50,2.3.52, 2.3.60) to /opt/so/saltstack/local/salt/nginx/etc then ran the command so-nginx-restart and also had no luck fixing the error. Any suggestions on how to fix the issue? 

Thanks

**Comment by code-by-justin** [comment]
Anything in the console of your web browser? If you go to Kibana are there events in there?

My issue was elastic search did not like capital letters in the username.

![image](https://user-images.githubusercontent.com/24820491/129412547-ad19c700-059c-40c7-9bd8-fee73118189c.png)
![image](https://user-images.githubusercontent.com/24820491/129412566-9eb69e8c-bc7a-4e30-9fbe-3d1b5cc94d00.png)

I just created a new users without capitals and my issue resolved itself.

**Reply by marianoka** [reply]
Hi!! The problem is your nginx.conf file. I suggest copy replace or review the file.

**Reply by dougburks** [reply]
@JJDsource Thanks for reporting this case sensitivity issue. I've created Issue https://github.com/Security-Onion-Solutions/securityonion/issues/5154 for this.
